pkg/tarball/write.go

```diff
@@ -41,7 +42,41 @@ func (ctx *Context) writeArchiveFromFS(dst io.Writer, fsys fs.FS) error {
 	return ctx.writeTar(tw, fsys)
 }
 
+func hasHardlinks(fi fs.FileInfo) bool {
```

Comment:
Expanding on my other comment, I'm wondering if we should have these functions live in a FileInfo type in apko/fs and to keep all the file logic constrained to that package.

Reply:
yes feel free to refactor this is just a hotfix :)